Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/BayesQuasi.py

```diff
@@ -407,7 +403,377 @@ def PyExec(self):
         if self._program == 'QL':
             self.setProperty('OutputWorkspaceProb', probWS)
 
+    def QLAddSampleLogs(self, workspace, fit_program, e_range, binning):
```

Comment:
might be best to rename this function to be more pythonic and `QL` prefix is no longer required e.g. `def _add_sample_logs(self, ...)`